tests/test_grid.py

```diff
@@ -442,14 +442,20 @@ def test_save_uniform(extension, binary, tmpdir):
 def test_grid_points():
     """Test the points methods on UniformGrid and RectilinearGrid"""
     # test creation of 2d grids
-    x_surf = y_surf = range(3)
-    z_surf = np.ones(3)
+    x = y = range(3)
+    z = [0,]
+    xx, yy, zz = np.meshgrid(x, y, z, indexing='ij')
+    points = np.c_[xx.ravel(order='F'), yy.ravel(order='F'), zz.ravel(order='F')]
     grid = pyvista.UniformGrid()
-    grid.points = np.array([x_surf, y_surf, z_surf]).transpose()
+    with pytest.raises(AttributeError):
+        grid.points = points
+    grid.origin = (0.0, 0.0, 0.0)
+    grid.dimensions = (3, 3, 1)
+    grid.spacing = (1, 1, 1)
     assert grid.n_points == 9
     assert grid.n_cells == 4
+    assert np.allclose(grid.points, points)
     del grid
```

Comment:
Curious, why are we deleting grid here?  If we want to check for garbage collection we might want a separate test.

Reply:
I had thought the same thing when updating this test... I’m not sure why. I will remove this

Garbage collection should be all good as far as these copies of the points go from what I can tell